corehq/blobs/migrate.py

```diff
@@ -395,7 +395,7 @@ def get_doc_migrator(self, filename):
             get_type_code=self.get_type_code,
         )
 
-    def get_document_provider(self):
+    def get_document_provider(self, **kwargs):
```

Comment:
```suggestion
    def get_document_provider(self, date_range=None):
        if date_range is not None:
            raise ValueError("{} does not support date_range".format(type(self).__name__))
```

Reply:
Alternately, and more simply, could just skip accepting `**kw` here. The error generated would be a little more generic.
```suggestion
    def get_document_provider(self):
```

Reply:
Probably my fault for suggesting this in the first place.